app/views/admin/api/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="admin">
+  <h2 class="heading"><%= ts('API Tokens') %></h2>
+
+  <ul class="navigation actions" role="navigation">
+    <li><%= span_if_current ts('Create a Token'), new_admin_api_path %></li>

```

Comment:
We usually say "New #{Thing}," e.g. "New Token," and we like the navigation to say what page you're on, so this could also use an "API Tokens" span_if_current. Also, it would be great if you could add our standard HTML comments in here:

```
<div class="admin">
  <!--Descriptive page name, messages and instructions-->
  <h2 class="heading"><%= ts("API Tokens") %></h2>
  <!--/descriptions-->

  <!--subnav-->
  <ul class="navigation actions" role="navigation">
    <li><%= span_if_current ts("API Tokens"), admin_api_path %></li>
    <li><%= span_if_current ts("New Token"), new_admin_api_path %></li>
  </ul>
  <!--/subnav-->

  <!--main content-->
  TABLE CODE
  <!--/content-->
</div>
```